src/Compilers/CSharp/Portable/Symbols/TypeWithAnnotations.cs

```diff
@@ -8,6 +8,7 @@
 using System.Linq;
 using System.Threading;
 using Microsoft.CodeAnalysis.PooledObjects;
+using System.ComponentModel;
```

Comment:
Nit: Not sure if you care about the order of usings, but if you do, consider sorting these